sourcecode/apis/contentauthor/database/factories/H5PLibraryFactory.php

```diff
@@ -18,7 +18,7 @@ public function definition(): array
             'fullscreen' => true,
             'embed_types' => 'div',
             'semantics' => '[]',
-            'tutorial_url' => 'http://burgerking.com',
+            'tutorial_url' => 'https://burgerking.com',
```

Comment:
lol

Reply:
No my link, I just made it "secure" to make PHPStorm happy. It complained about "Link with unencrypted protocol" and "HTTP links are not secure"